packages/funman/tasks/validate_modelconfig.py

```diff
@@ -0,0 +1,108 @@
+import sys
+import os
+import json
+from core.taskrunner import TaskRunnerInterface
+
+# Funman imports
+from funman import Funman
+from funman.config import FUNMANConfig
+from funman.model.model import _wrap_with_internal_model
+from funman.scenario.scenario import AnalysisScenario
+from funman.model.petrinet import PetrinetModel
+from funman.representation.parameter_space import ParameterSpace
+from funman.server.query import (
+    FunmanProgress,
+    FunmanResults,
+    FunmanWorkUnit,
+)
+import pydantic
+from pydantic import TypeAdapter
+from funman.model.generated_models.petrinet import Model as GeneratedPetrinet
+
+
+# FIXME
+dummy_id = "xyz"
+
+adapter = TypeAdapter(GeneratedPetrinet)
+current_results = None
+
+def cleanup():
+    print("Task cleanup")
+
+def run_validate(model: PetrinetModel, request):
+    current_results = FunmanResults(
+        id=dummy_id,
+        model=model,
+        request=request,
+        parameter_space=ParameterSpace(),
+    )
+
+    # Update callback
+    def update_current_results(scenario: AnalysisScenario, results: ParameterSpace) -> FunmanProgress:
+        progress = current_results.update_parameter_space(scenario, results)
+        print("update hook", progress)
+
+
+    # Invoke solver
+    work = FunmanWorkUnit(id=dummy_id, model=model, request=request)
+    f = Funman()
+    scenario = work.to_scenario()
+    config = (
+        FUNMANConfig()
+        if work.request.config is None
+        else work.request.config
+    )
+    result = f.solve(
+        scenario,
+        config=config,
+        # haltEvent=self._halt_event,
+        resultsCallback=lambda results: update_current_results(scenario, results),
+    )
+    print("Done solver portion")
+    current_results.finalize_result(scenario, result)
+    print(current_results.model_dump_json(by_alias=False))
+
+    return current_results.model_dump_json(by_alias=False)
+
+
+def taskrunner_wrapper():
+    print("Taskrunner wrapper")
+    try:
+        taskrunner = TaskRunnerInterface(description="AMR to MMT")
```

Comment:
I think the description needs to be updated

Reply:
thx